cms/templates/home_page.html

```diff
@@ -10,9 +10,16 @@
 {% endblock %}
 
 {% block content %}
-  <h1>{{ page.title }}</h1>
-  <p>
-    Image:<br />
-    <img src="{% wagtail_img_src page.hero %}" alt="home" />
-  </p>
+    <div class="hero-content" style="background-image: url('{% wagtail_img_src page.hero%}')">
+      <div class="container hero-container">
```

Comment:
Need to switch these classes to make the background image stretch correctly

```suggestion
    <div class="hero-container" style="background-image: url('{% wagtail_img_src page.hero%}')">
      <div class="container hero-content">
```